src/index.js

```diff
@@ -65,53 +66,67 @@ module.exports = {
     // If we are on Netlify pull the secrets that have been added through the UI
     // so that we can contextualize them as well.
     if (isNetlify) {
-      console.log('Adding Netlify Build secrets to secrets array');
       secrets = { ...secrets, ...netlifyConfig.build.environment };
+
+      // Ensure that dotenv gets pushed to the external bundle.
+      if (!Array.isArray(netlifyConfig.functions['*'].external_node_modules)) {
+        netlifyConfig.functions['*'].external_node_modules = [];
+      }
+      netlifyConfig.functions['*'].external_node_modules.push('dotenv');
```

Comment:
Ensures that dotenv is bundled with our LAMBDAs, especially when bundling with esbuild.